README.md

```diff
@@ -17,6 +17,18 @@ To download the tool you have to have Node.js installed on your machine. After y
 $ npm install -g speedo
 ```
 
+## Update
+
+```sh
+$ speedo --version
+// => 1.2.2
+
+$ npm install -g speedo@latest
```

Comment:
What happens if someone uses a local Speedo installation?

Reply:
I think then they will know what to do :) We have an installation step above with `-g` so we keep being consistent here.